src/Codor/Sniffs/Classes/FinalPrivateSniff.php

```diff
@@ -119,7 +119,7 @@ protected function handleErrors($phpcsFile, $stackPtr)
         }
 
         foreach ($this->protectedVariableTokens as $protectedVariableToken) {
-            $this->handleProtectedVariableToken($protectedVariableToken, $phpcsFile, $stackPtr);
+            $this->handleProtectedVariableToken($protectedVariableToken, $phpcsFile);
```

Comment:
Since `$stackPtr` is not used anymore in this method it can be removed from the parameters and the PHPDoc.

Reply:
Done 😃

Reply:
Thank you for your reactivity!
The call to `handleErrors()` needs to be modified too.

Reply:
Jumping right onto!